images/router/haproxy/conf/haproxy-config.template

```diff
@@ -425,6 +424,9 @@ frontend fe_sni
 ##########################################################################
 # backend for when sni does not exist, or ssl term needs to happen on the edge
 backend be_no_sni
+        {{- with $value := maxTimeoutFirstMatchedAndClipped .State "haproxy.router.openshift.io/timeout" $timeSpecPattern (env "ROUTER_DEFAULT_SERVER_TIMEOUT") "30s"  }}
```

Comment:
Why do you call the special `maxTimeoutFirstMatchedAndClipped` here but continue to call `clipHAProxyTimeoutValue` in the other places?

Reply:
We want [the maximum of all the backend timeouts](https://github.com/openshift/router/pull/536#issuecomment-1855963547) to be set on the middle backends (`be_sni`, `be_no_sni`). Otherwise we get [the warning during the config reloading](https://github.com/openshift/router/pull/536#issuecomment-1834660723).